1-exercises/E-for-loop/exercise2.js

```diff
@@ -27,7 +27,9 @@ const AGES = [
 ];
 
 // TODO - Write for loop code here
-
+for (let i=0;i<WRITERS.length-1;i++){
```

Comment:
as you're starting from 0 and just using a `<` instead of `<=`, you just need to check if `i<WRITERS.length` instead of .length-1